parsimonious/tests/test_nodes.py

```diff
@@ -142,3 +142,8 @@ def visit_greeting(self, thing, visited_children):
             raise PrimalScream('This should percolate up!')
 
     assert_raises(PrimalScream, Screamer().parse, 'howdy')
+
+
+def test_node_inequality():
+    node = Node('text', 'o hai', 0, 5)
+    assert node != 5
```

Comment:
For consistency and to work properly under `python -O` (rather than getting optimized out), let's import `ok_` from `nose.tools` and do `ok_(node != 5)`. Since we're removing the explicit `None` checking above, it might also not be a bad idea to test equality against `None` if we're not doing it elsewhere.